ArduPlane/is_flying.cpp

```diff
@@ -270,6 +278,12 @@ void Plane::crash_detection_update(void)
         crash_state.checkedHardLanding = false;
     }
 
+    // if we have no GPS lock and we don't have a functional airspeed
+    // sensor then don't do crash detection
+    if (gps.status() < AP_GPS::GPS_OK_FIX_3D && (!airspeed.use() || !airspeed.healthy())) {
```

Comment:
I don't like how this allows the crash timers to be set above, then just clears the flag. I'm OK with disabling the crash detector in this scenario but it would better be done at the very top before we bother checking the other stuff